DependencyInjection/Configuration.php

```diff
@@ -81,6 +79,8 @@ public function getConfigTreeBuilder()
                             ->scalarNode('command')->defaultValue(null)->end()
                             ->scalarNode('connection')->defaultValue(null)->end()
                             ->scalarNode('queue')->defaultValue(null)->end()
+                            ->booleanNode('exclusive_processor_stack')->defaultValue(false)->end()
+                            ->append($this->addProcessorStack())
```

Comment:
@stof I need your help on this one, I added fixXmlConfig as you requested, but it breaks test.
I read the doc, but really, I didn't understand where to put this and why it's break test.